sdk/policyinsights/Azure.ResourceManager.PolicyInsights/src/autorest.md

```diff
@@ -8,8 +8,8 @@ azure-arm: true
 csharp: true
 library-name: PolicyInsights
 namespace: Azure.ResourceManager.PolicyInsights
-require: https://github.com/Azure/azure-rest-api-specs/blob/aa8a23b8f92477d0fdce7af6ccffee1c604b3c56/specification/policyinsights/resource-manager/readme.md
-tag: package-2022-03
+require: https://github.com/Azure/azure-rest-api-specs/blob/05a9cdab363b8ec824094ee73950c04594325172/specification/policyinsights/resource-manager/readme.md
```

Comment:
Please update the CHANGELOG file to specify the release day for this new package